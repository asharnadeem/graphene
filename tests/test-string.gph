int main() {
    list<string> l;
    l.push_back("1");
    l.push_back("2");
    l.push_back("3");
    while(l.size > 0) {
        prints(l.pop_back());
    }
    string s = "hello";
    prints(s);
    node<string> n;
    n.val = "world";
    prints(n.val);
}