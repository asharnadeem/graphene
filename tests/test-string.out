3
2
1
hello
world
